spec/models/miq_product_feature_spec.rb

```diff
@@ -1,15 +1,16 @@
 require "spec_helper"
 
 describe MiqProductFeature do
-  before do
-    @expected_feature_count = 846
+  before(:all) do
```

Comment:
I like what this change is doing but before(:all) is usually the :hammer: when we need to change what we're testing.  While this version of a before(:all) is correct, namely it manually rollbacks it's changes, the assumption is that seed is doing the right thing and didn't skip any product features.

A hardcoded count is clearly brittle.

Can we instead read the yml and count the number of identifier keys in the deeply nested hash and use that number?  I believe that key is supposed to be unique.  After seeding, that number of rows should exist in the table.

With that tested by itself, the others don't need to know that number.